data/coalition/coalition.txt

```diff
@@ -611,6 +611,21 @@ phrase "heliarch"
 		"Unwavering"
 		"Obdurate"
 		"Unbending"
+		"Resolute"
+		"Steadfast"
+		"Ardent"
+		"Undefeated"
+		"Enduring"
+		"Impervious"
+		"Honorable"
+		"Brave"
+		"Skilled"
+		"Stoic"
+		"Imposing"
```

Comment:
```suggestion
		"Resilient"
		"Devout"
		"Devoted"
```
Imposing feels a bit on the nose, so I suggest these as options, and looks like you've already replaced Tough with Stoic which seems good.